### PR TITLE
[1778] Add setup info for Linux

### PR DESCRIPTION
**Pull request by faissalMT**
### Context
I had a little trouble setting up, this should avoid that in future.  

### Changes proposed in this pull request
Mention the need to start PostgreSQL.  

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by faissalMT** [comment]
[Add documentation for Linux setup](https://trello.com/c/YEyiq45Z/1778-add-documentation-for-linux-setup)

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 ### Native
 
+0. Linux users: You may need to manually start PostgreSQL. Most major distributions use systemd as their init so all you need to do is run `systemctl start postgresql`
```
Comment (misaka) [inline README.md]: What if we add a pre-amble to this section that points people to https://gorails.com/setup to get them 90% of the way there? There may be some discrepencies, but it seems like an easy way to help people get started.

File: README.md [hunk 2]
```diff
@@ -172,7 +173,7 @@ complex operations that just aren't already packaged up in the app.
 
 The script's functionality is accessed using sub-commands with built-in
 documentation. This is the best way to discover it's functionality and the
-commands available, and is accessable with the `--help` option:
+commands available, and is accessible with the `--help` option:
```
Comment (timabell) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -21,6 +21,7 @@
 
 ### Native
 
+0. If you haven't already, follow this [tutorial](https://gorails.com/setup) to setup your Rails environment, make sure to install PostgreSQL 9.6 as the database
```
Comment (timabell) [inline README.md]: Starting at zero is such a linux/stallman thing to do :laughing:

Comment (timabell) [reply README.md]: Also today I learned some html

Selection_707

**Pull request merged**